src/drawers/areaDrawer.ts

```diff
@@ -8,6 +8,13 @@ export module _Drawer {
     private _areaSelection: D3.Selection;
     private _drawLine = true;
 
+    constructor(key: string, interpolationMode?: string) {
+      super(key);
+      if (interpolationMode !== null) {
```

Comment:
We don't need this check if we start with a interpolationMode and we assume that we always have a valid interpolationMode